airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -48,6 +54,101 @@
 )
 
 
+class DataProcCreateWorkflowBaseOperator(BaseOperator):
```

Comment:
```suggestion
class DataprocCreateWorkflowBaseOperator(BaseOperator):
```

For consistency with other dataproc-related classes classes